contributing.md

```diff
@@ -0,0 +1,27 @@
+# How To Become A Contributor And Submit Your Own Code
+There are few guidelines that we need contributors to follow so that we can have a chance of keeping on top of things.
+
+## Getting Started
+This repo uses Lerna(https://lernajs.io/) as a top level setup, which is a tool that optimizes the workflow around managing multi-package repositories with git and npm.
+
+* Make sure you have forked the desired repo(https://github.com/electrode-io/electrode.git) for developing and testing your code changes.
+
+* Make sure you have installed Lerna
+
+````
+$ npm install --global lerna
+$ lerna init
+````
+
+* Run `lerna bootstrap` for bootstrapping the packages in the current Lerna repo. It will install all their dependencies and linking any cross-dependencies.
+
+* Run `gulp samples-local` for pulling electrode packages from local under `/packages` folder.
+
+* Now you can go to the `samples/` folder, pick or create any samples, develop and test your changes over there.
```

Comment:
give some instructions on how to run sample in `dev` mode, try it out, see it in browser, etc.